keyboards/naked60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,151 @@
+       KC_ESC,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12,
+  //|--------+--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------+--------|
+      KC_TILD,    KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                      KC_6,    KC_7,    KC_8,    KC_9,    KC_0,  KC_DEL,
+  //|--------+--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------+--------|
+       KC_DEL,   KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,                     KC_F6, KC_MINS,  KC_EQL, KC_LBRC, KC_RBRC, KC_BSLS,
+  //|--------+--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------+--------|
+        _____,   KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,                    KC_F12, KC_NUHS, KC_NUBS,   _____,   _____,   _____,
+  //|--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+                 _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____ 
+          //`------------------------------------------------------------------------------------------------------------'
+  ),
+
+
+  [_ADJUST] = LAYOUT( /* Base */
+  //,-----------------------------------------------------|                 |-----------------------------------------------------.
+       KC_ESC,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,   KC_F9,  KC_F10,  KC_F11,  KC_F12,
+  //|--------+--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------+--------|
+        _____,   RESET,   XXXXX,   XXXXX,   XXXXX,   XXXXX,                     XXXXX, KC_BTN1, KC_MS_U, KC_BTN2,   XXXXX,LALT(KC_PSCR),
+  //|--------+--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------+--------|
+        _____,   XXXXX,   XXXXX,   XXXXX,   XXXXX,   XXXXX,                     XXXXX, KC_MS_L, KC_MS_D, KC_MS_R,   XXXXX, KC_PSCR,
+  //|--------+--------+--------+--------+--------+--------|                 |--------+--------+--------+--------+--------+--------|
+        _____,   XXXXX,   XXXXX,   XXXXX,   XXXXX,   XXXXX,                     XXXXX,   XXXXX,   XXXXX,   XXXXX,   XXXXX,LCA(KC_DEL),
+  //|--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------|
+                 _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____,   _____
+          //`------------------------------------------------------------------------------------------------------------'
+  )
+};
+
+static inline void update_change_layer(bool pressed, uint8_t layer1, uint8_t layer2, uint8_t layer3) {
```

Comment:
You could probably simply all of this by using `MO(_LAYER)` and using the `update_tri_layer_state` function. 
https://docs.qmk.fm/#/ref_functions?id=update_tri_layer_statestate-x-y-z

Reply:
I'm sorry.
I tried it but couldn't.
I want to move to the Adjust layer both when I press the Adjust key and when I press the Lower and RAISE keys.